TicTacToe.cpp

```diff
@@ -77,3 +78,36 @@ int TicTacToe::checkVictoryHorizontal(int teamSide)
     }
     return point;
 }
+
+
+void TicTacToe:viewAll()
+{
+	int hold[26];   \\ create array to hold field integers 
+	int count = 0;  \\ create counter to store integers in the array
```

Comment:
I don't see where this is ever incremented. Granted, I haven't tried running this code, but I think it would just keep overwriting the zero-th location in the `hold` array.

Reply:
You're correct- I forgot to add the incrementation. I will adjust it shortly.